build_linux.sh

```
@@ -59,25 +53,11 @@ if (echo "$_OS" | grep -q "Linux.*") ; then
     fi
 fi
 
+JP_APP_VERSION='99.9.9'
```

Shellcheck (0.9.0-1)

[warning] JP_APP_VERSION appears unused. Verify use (or export if used externally).

(SC2034)